docker/Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:jammy
+
+RUN apt-get update && \
+  DEBIAN_FRONTEND='noninteractive' \
+  DEBCONF_NONINTERACTIVE_SEEN='true' \
+  apt-get install --yes \
+    gfortran \
+    build-essential \
+    git \
+    curl \
+    libblas-dev \
+    liblapack-dev \
+    autoconf \
+    automake \
+    autoconf-archive \
+    libpython2.7-dev \
+    libpython2.7 \
+    python2.7 \
+    libtool \
+    libc6-dev \
+    mpich \
+    sudo \
+    nano
+
+RUN useradd \
+  --create-home \
+  lithomop_user && echo "lithomop_user:password" | chpasswd
+
+
+RUN sudo adduser lithomop_user sudo
+
+# Need to create this link so that pythia will compile
+RUN ln -s /usr/bin/python2.7 /usr/bin/python;
+
+WORKDIR /home/lithomop_user
+
+USER lithomop_user
+
+#apt will install pip3, so need to do this to install pip 2.7
+RUN curl -L -O "https://bootstrap.pypa.io/pip/2.7/get-pip.py"; echo "password" | sudo -S python2.7 get-pip.py; pip install setuptools; pip install ez_setup;
+
+
+ENV PETSC_DIR="/home/lithomop_user/petsc-2.3.3-p16"
+ENV PETSC_ARCH="/home/lithomop_user/petsc-2.3.3-p16/lib/linux-gnu-c-debug"
+ENV PATH="/home/lithomop_user/petsc-2.3.3-p16/externalpackages/mpich2-1.0.5p4/bin:${PATH}"
+
+RUN curl -L -O "https://web.cels.anl.gov/projects/petsc/download/release-snapshots/petsc-2.3.3.tar.gz"; tar -xvf petsc-2.3.3.tar.gz; cd petsc-2.3.3-p16; mkdir externalpackages; cd externalpackages; curl -L -O "https://www.mpich.org/static/downloads/1.0.5p4/mpich2-1.0.5p4.tar.gz"; tar -xvf mpich2-1.0.5p4.tar.gz; cd mpich2-1.0.5p4; ./configure; echo "password" | sudo -S make all; echo "password" | sudo -S make install; 
```

Comment:
Do we really need `petsc-2.3.3` (released in 2007) or can we use a later version?

Reply:
I'd suggest removing the manual download of mpich and instead use the system version from apt.